src/Microsoft.Identity.Client/Exceptions/CoreErrorMessages.cs

```diff
@@ -145,5 +145,11 @@ public static string InvalidRedirectUriReceived(string invalidRedirectUri)
         {
             return $"Invalid RedirectURI was received ({invalidRedirectUri})  Not parseable into System.Uri class.";
         }
+
+        public const string TelemetryClassIsObsolete =
+            "Telemetry is now specified per ClientApplication.  See https://aka.ms/msal-net-application-configuration";
```

Comment:
See https://aka.ms/msal-net-3-breaking-changes and https://aka.ms/msal-net-application-configuration